# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_brotli_http.js
# Commit: 99e71b905661
# Full Hash: 99e71b9056618a081f8bc2fb7803ba208db3a5b0
# Author: Randell Jesup <rjesup@mozilla.com>
# Date: 2024-04-10 09:45:41
# Regressor Bug: 1871963
# File Overlap Count: 1
# Description:
#   Bug 1871963: Implement zstd content-encoding support r=necko-reviewers,valentin,devtools-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D205109
# ==============================================================================

diff -r 68f119112940 -r 99e71b905661 netwerk/test/unit/test_brotli_http.js
--- a/netwerk/test/unit/test_brotli_http.js	Tue Apr 09 15:26:45 2024 +0000
+++ b/netwerk/test/unit/test_brotli_http.js	Tue Apr 09 15:26:46 2024 +0000
@@ -100,7 +100,7 @@
     });
     equal(
       Services.prefs.getCharPref("network.http.accept-encoding.secure"),
-      "gzip, deflate, br"
+      "gzip, deflate, br, zstd"
     );
     let { req, buff } = await new Promise(resolve => {
       let chan = NetUtil.newChannel({